packages/pdf/pyproject.toml

```diff
@@ -25,6 +25,8 @@ classifiers = [
 	"Programming Language :: Python :: 3.9",
 ]
 
+include = ["*.libspec"]
```

Comment:
@cmin764 I at some point tried adding these locally and didn't actually find any libspec files in the built wheels, do you know if they need a separate command (does something like `inv code.build` handle them?), they might also be missing from RPA.Assistant because I have been running `poetry build` locally...

Reply:
Yes, you need to build/deploy with our `inv` commands, otherwise you'll lose the libspec files. (and maybe other settings in general)